test/IntegrationTests/Helpers/TestHelper.cs

```diff
@@ -66,7 +66,7 @@ public string GetTestAssemblyPath()
     /// and returns the Process instance for further interaction.
     /// </summary>
     /// <returns>Test application process</returns>
-    public Process StartTestApplication(int traceAgentPort = 0, int metricsAgentPort = 0, int logsAgentPort = 0, string arguments = null, string packageVersion = "", int aspNetCorePort = 0, string framework = "", bool enableStartupHook = true, bool enableClrProfiler = true)
+    public Process StartTestApplication(int traceAgentPort = 0, int otlpTraceCollectorPort = 0, int metricsAgentPort = 0, int logsAgentPort = 0, string arguments = null, string packageVersion = "", int aspNetCorePort = 0, string framework = "", bool enableStartupHook = true, bool enableClrProfiler = true)
```

Comment:
tracerAgentPort -> tracerZipkinAgentPort
otlpTraceCollectorPort -> tracerAgentPort 

What is more, I would move tracerZipkinAgentPort at postion 4 o even later.

Reply:
I have not touched it as the end goal is to:

- otlpTraceCollectorPort  -> tracerAgentPort
- remove tracerZipkinAgentPort